tests/test_text_generation_example.py

```diff
@@ -159,58 +154,58 @@ def _test_text_generation(
     if fp8:
         if "--trim_logits" not in command:
             command += ["--trim_logits"]
-        if "Llama-2" in model_name or "Mistral" in model_name:
-            command.remove("--max_new_tokens 100")
+        if "Llama-2" in model_name:
+            command.insert(-2, "--use_flash_attention")
+            command.insert(-2, "--flash_attention_recompute")
+            command.insert(-2, "--bucket_size 128")
+            command.insert(-2, "--bucket_internal")
+        elif "falcon-180b" in model_name.lower():
+            command.insert(-2, "--flash_attention_recompute")
+
+        global prev_quant_model_name
+        global prev_quant_rank
+        measure_command = None
+        # FP8 Measurement only needed
+        if (not prev_quant_model_name) or (prev_quant_model_name != model_name) or (prev_quant_rank != world_size):
+            measure_command = [
+                x for x in command if not x.startswith("--max_new_tokens")
+            ]  # Remove max_new_tokens for mesurement
+            measure_command = [
+                x if not x.startswith("--batch_size") else "--batch_size 1" for x in measure_command
+            ]  # Remove batch_size for mesurement
+
+            prev_quant_model_name = model_name
+            prev_quant_rank = world_size
+
+        # FP8 text generation
+        command += [
+            f"--max_input_tokens {max_input_tokens}",
+            "--limit_hpu_graphs",
+        ]
 
     with TemporaryDirectory() as tmp_dir:
         command.append(f"--output_dir {tmp_dir}")
-        print(f"\n\nCommand to test: {' '.join(command)}\n")
-
         command.append(f"--token {token.value}")
 
         pattern = re.compile(r"([\"\'].+?[\"\'])|\s")
 
         if fp8:
-            env_variables["QUANT_CONFIG"] = os.path.join(
-                path_to_example_dir, "text-generation/quantization_config/maxabs_measure_include_outputs.json"
-            )
-            command = [x for y in command for x in re.split(pattern, y) if x]
-            subprocess.run(command, env=env_variables)
+            env_variables["TQDM_DISABLE"] = "1"
+            if measure_command:
+                measure_command.append(f"--token {token.value}")
+                env_variables["QUANT_CONFIG"] = os.path.join(
+                    path_to_example_dir, "text-generation/quantization_config/maxabs_measure_include_outputs.json"
+                )
+                measure_command = [x for y in measure_command for x in re.split(pattern, y) if x]
+                print(f"\n\nMeasure Command to test: {' '.join(measure_command)}\n")
```

Comment:
```suggestion
                print(f"\n\nMeasure Command to test: {' '.join(measure_command[:-2])}\n")
```
We need to avoid printing the token to prevent a leak.